<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaserSettings</class>
 <widget class="QDialog" name="LaserSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>145</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="LASER_RADIO_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string># 1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="LASER_RADIO_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string># 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ENLARGE_BTN">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NARROW_BTN">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RIGHT_BTN">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/directions/right</normaloff>:/directions/right</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="LEFT_BTN">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/directions/left</normaloff>:/directions/left</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="UP_BTN">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/directions/up</normaloff>:/directions/up</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="DOWN_BTN">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/directions/down</normaloff>:/directions/down</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
